config/default.yml

```diff
@@ -65,6 +65,16 @@ Rails:
   Enabled: true
   DocumentationBaseURL: https://docs.rubocop.org/rubocop-rails
 
+Rails/ActionControllerFlashBeforeRender:
+  Description: 'Use `flash.now` instead of `flash` before `render`.'
+  StyleGuide: 'https://rails.rubystyle.guide/#flash-before-render'
+  Reference: 'https://api.rubyonrails.org/classes/ActionController/FlashBeforeRender.html'
+  Enabled: 'pending'
+  SafeAutocorrect: true
+  VersionAdded: '2.15'
+  Include:
+    - '**/test/**/*.rb'
```

Comment:
Is this `Include` configuration necessary?

Reply:
It is not. 😁